blog/en/blog/2023/01/18/consul-with-apisix.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "How to Integrate API Gateway and Consul? Not Consul K/V"
+authors:
+  - name: "Yihao LI"
+    title: "Author"
+    url: "https://github.com/Fabriceli"
+    image_url: "https://github.com/Fabriceli.png"
+  - name: "Yihao LI"
+    title: "Technical Writer"
+    url: "https://github.com/Fabriceli"
+    image_url: "https://github.com/Fabriceli.png"
```

Comment:
you can delete this part, only keep "Author". Cause you are the author of this article, and no one else edits this article except you. such as this:(same with ZH)
<img width="528" alt="image" src="https://user-images.githubusercontent.com/39793568/213342198-dd5153ab-714c-4cb7-91c1-a3d780825ddb.png">

Reply:
done